src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -77,12 +100,30 @@ export default {
             imageFormat: '',
             imageHistory: [],
             imageUrl: '',
+            age: '',
             isPaused: false,
             metadata: {},
             requestCount: 0,
-            timeFormat: ''
+            time: undefined,
+            timeFormat: '',
+            clock: clock,
+            ageTracker: undefined,
+            refreshCSS: false
         };
     },
+    computed: {
+        isImageNew() {
+            let cutoff = FIVE_MINUTES;
+            let age = this.numericImageAge();
```

Comment:
See note elsewhere about making `numericImageAge` a property of `this`. That way it only needs to be called once, and it will be possible for Vue to optimize calls to `isImageNew`.

Reply:
Set numericImageAge to computed value